src/main/java/core/basesyntax/service/impl/operation/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.model.Fruit;
+
+public interface OperationHandler {
+    int MIN_FRUIT_QUANTITY = 0;
+    boolean doOperation(Fruit fruit, int quantity);
```

Comment:
According to the SOLID principles, specifically the Single Responsibility Principle, an interface should be focused on a single functionality. The name `OperationHandler` suggests that it should handle an operation, but the method `doOperation` is quite generic. Consider renaming the method to more accurately describe its purpose, such as `applyOperation`, and ensure that the implementing classes follow this contract clearly.